lib/fathead/android_api_docs/parse.py

```diff
@@ -128,8 +128,8 @@ def build_article(soup, filePath, fileName):
 
         #Build the abstract from the description and example code usage
         abstract = ""
-        abstract += (classDescription + "\\n").encode('utf-8')
-        abstract += "<pre><code>%s</code></pre>" % ((firstCode + "\\n" + secondCode).encode('utf-8'))
+        abstract += (classDescription).encode('utf-8')
+        abstract += "<pre><code>%s</code></pre>" % ((firstCode + "\n" + secondCode).encode('utf-8'))
```

Comment:
@TM37 this is incorrect. Your abstracts now have literal newlines in them. Each entry in the output.txt should only be 1 line. Any newlines in the abstract should be escaped 👍